src/App.js

```diff
@@ -1,13 +1,80 @@
-import Navbar from "./Navbar";
 import GlobalStyle from "./globalStyles";
+import Navbar from "./Navbar";
+import Homapage from "./Homapage";
+import Profile from "./Profile";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+
+import { initializeApp } from "firebase/app";
+import { getStorage } from "firebase/storage";
+import {
+  getFirestore,
+  collection,
+  onSnapshot,
+  setDoc,
+  doc,
+  Timestamp,
+  getDocs,
+  getDoc,
+  addDoc,
+  deleteDoc,
+} from "firebase/firestore";
+
+// TODO: Add SDKs for Firebase products that you want to use
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+// Your web app's Firebase configuration
+// For Firebase JS SDK v7.20.0 and later, measurementId is optional
+const firebaseConfig = {
+  apiKey: "AIzaSyAhSbQw18_zFQ7GzjqhjeAdNeDYl10udoY",
```

Comment:
apiKey should not be put here immediately. It is extremely dangerous.
Please new a config file. And just put it in your own local env, which means adding this file to .gitignore.